recipes/deadlink/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "deadlink" %}
+{% set version = "0.5.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deadlink-{{ version }}.tar.gz
+  sha256: a0fb4855994997b323f684e9022c2bef68980852ab1c4374ea0c4e508dd75e7d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```
This will be done by conda-forge.

---

Done.